lecture10/template/cfn_network.yaml

```
@@ -0,0 +1,160 @@
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-igw"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+  CfnInternetGatewayAttachment:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      InternetGatewayId: !Ref CfnInternetGateway
+      VpcId: !Ref CfnVPC
+
+  # ------------------------------------------------------------#
+  #  Public Subnet
+  # ------------------------------------------------------------#
+  CfnPublicSubnet1A:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      CidrBlock: 10.0.0.0/20
+      VpcId: !Ref CfnVPC
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-public-subnet-1a"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+  CfnPublicSubnet1C:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      CidrBlock: 10.0.16.0/20
+      VpcId: !Ref CfnVPC
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-public-subnet-1c"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+  CfnPrivateSubnet1A:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      CidrBlock: 10.0.128.0/20
+      VpcId: !Ref CfnVPC
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-private-subnet-1a"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+  CfnPrivateSubnet1C:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      CidrBlock: 10.0.144.0/20
+      VpcId: !Ref CfnVPC
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-private-subnet-1c"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+
+  # ------------------------------------------------------------#
+  #  RouteTable　
+  # ------------------------------------------------------------#
+  CfnPublicRouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref CfnVPC
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-public-route-table"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+  CfnPrivateRouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref CfnVPC
+      Tags:
+        - Key: Name
+          Value: !Sub "${PJName}-private-route-table"
+        - Key: !Ref TagKey
+          Value: !Ref PJName
+
+  PublicRoute:
+    Type: AWS::EC2::Route
+    Properties:
+      RouteTableId: !Ref CfnPublicRouteTable
+      DestinationCidrBlock: "0.0.0.0/0"
+      GatewayId: !Ref CfnInternetGateway
+
+  CfnPublicSubnetRouteTableAssociation1A:
+    Type: AWS::EC2::SubnetRouteTableAssociation
+    Properties:
+      SubnetId: !Ref CfnPublicSubnet1A
+      RouteTableId: !Ref CfnPublicRouteTable
+  CfnPublicSubnetRouteTableAssociation1C:
+    Type: AWS::EC2::SubnetRouteTableAssociation
+    Properties:
+      SubnetId: !Ref CfnPublicSubnet1C
+      RouteTableId: !Ref CfnPublicRouteTable
```

```suggestion
  CfnPublicRouteTable:
    Type: AWS::EC2::RouteTable
    Properties:
      VpcId: !Ref CfnVPC
      Tags:
        - Key: Name
          Value: !Sub "${PJName}-public-route-table"
        - Key: !Ref TagKey
          Value: !Ref PJName
          
  CfnPrivateRouteTable:
    Type: AWS::EC2::RouteTable
    Properties:
      VpcId: !Ref CfnVPC
      Tags:
        - Key: Name
          Value: !Sub "${PJName}-private-route-table"
        - Key: !Ref TagKey
          Value: !Ref PJName

  PublicRoute:
    Type: AWS::EC2::Route
    Properties:
      RouteTableId: !Ref CfnPublicRouteTable
      DestinationCidrBlock: "0.0.0.0/0"
      GatewayId: !Ref CfnInternetGateway

  CfnPublicSubnetRouteTableAssociation1A:
    Type: AWS::EC2::SubnetRouteTableAssociation
    Properties:
      SubnetId: !Ref CfnPublicSubnet1A
      RouteTableId: !Ref CfnPublicRouteTable
      
  CfnPublicSubnetRouteTableAssociation1C:
    Type: AWS::EC2::SubnetRouteTableAssociation
    Properties:
      SubnetId: !Ref CfnPublicSubnet1C
      RouteTableId: !Ref CfnPublicRouteTable
```
空行の使い方に一貫性をもたせましょう！（他の箇所も確認お願いします！）

---

修正しました。 13ef768825f35bcf45c442f132fcd2c4c7bd65e1